time_tracker.py

```diff
@@ -69,83 +45,25 @@ def convertTime(time):
-    if args.x == 'start':
-        if has_duplicate_task(data["tasks"], args.y):
-            if has_finish(data["tasks"][args.y],"finish"):
-                with open(filePath, 'w', encoding='utf-8') as f:
-                    data["tasks"][args.y].append({args.x: time.time()})
-                    json.dump(data, f, ensure_ascii=False, indent=2)
-
-            else:
-                sys.exit('You already started this project')
-
-        else:
-            with open(filePath, 'w', encoding='utf-8') as f:
-                dest = {
-                    args.y: [
-                        {
-                            args.x: time.time()
-                        }
-                    ]
-                }
-                data["tasks"].update(dest)
-                json.dump(data, f, ensure_ascii=False, indent=2)
-
-    elif args.x == 'finish':
-        if has_duplicate_task(data["tasks"], args.y):
-            if has_finish(data["tasks"][args.y],args.x):
-                sys.exit("You do not have unfinished project named " + args.y)
-            else:
-                print("adding finishing time")
-                with open(filePath, 'w', encoding='utf-8') as f:
-                    last_index = len(data["tasks"][args.y]) - 1
-                    data["tasks"][args.y][last_index][args.x] = time.time()
-                    json.dump(data, f, ensure_ascii=False, indent=2)
-
-        else:
-            sys.exit("You do not have unfinished project named " + args.y)
-
-    elif args.x == 'cancel':
-        if has_finish(data["tasks"][args.y],"finish") == False:
-            cancel_task(data["tasks"],args.y)
-            sys.exit("Project Not Found")
-        else:
-            sys.exit("Not Authorized")
-
-    elif args.x == 'get':
-        if has_finish(data["tasks"][args.y],"finish"):
-            timeDiff = 0
-            for task in data["tasks"][args.y]:
-                if datetime.fromtimestamp(task["start"]) >= Fromtimestamp and Totimestamp >= datetime.fromtimestamp(task["finish"]):
-                    startTime = task["start"]*1000
-                    finishTime = task["finish"]*1000
-                    timeDiff += finishTime - startTime
-            if timeDiff > 0:
-                sec, min, hr = convertTime(int(timeDiff))
-                print("{} hr {} min {} sec".format(hr, min, sec))
-            else:
-                sys.exit("You have no data available between two values")
-
-        else:
-            sys.exit("You have not finished project named " + args.y)
-
-
-else:
-    print("Either file is missing or is not readable, creating file...")
-    info = {
-        "tasks": {
-            args.y: [
-                {
-                    args.x: time.time()
-                }
-            ]
-        }
-    }
 
-    with open(filePath, 'w') as json_file:
-        json.dump(info, json_file, ensure_ascii=False, indent=2)
+task_manager = TaskManager()
+task_manager.parse_data(data)
+msg = "No Message Available"
```

Comment:
you don't need a default message here. one of `start`, `finish`, `cancel`, and `get` will be triggered, and there will always be a message.